app/scripts/components/common/loading-skeleton.tsx

```diff
@@ -128,6 +130,30 @@ const ContentLoadingSelf = styled.div`
   }
 `;
 
+const ChartLoadingWrapper = styled.div`
+  position:relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-flow: column;
+  text-align: center;
+  padding: ${variableGlsp()};
+  gap: ${glsp()};
+  aspect-ratio: 16/9;
+  color: ${themeVal('color.base-400')};
+
+  &::before {
+    position: absolute;
+    inset: 0;
+    display: flex;
+    content: '';
+    z-index: -1;
+    padding: ${variableGlsp()};
+    background: ${themeVal('color.base-100')};
+    animation: ${pulse} 0.8s ease 0s infinite alternate;
```

Comment:
💅 
Can we use the occasion to store the whole string in a const? (it's repeated 4 times)